ibis/expr/datatypes.py

```diff
@@ -1201,6 +1280,27 @@ def infer_null(value):
     return null
 
 
+# geo spatial data type
+@infer.register(tuple)
+def infer_point(value):
```

Comment:
We have not overloaded infer with tuple argument yet, so theoretically We could use this for Point datatype, but We might have a tuple type in the future. IMO remove this.